build_common/src/lib.rs

```diff
@@ -1,2 +1,4 @@
 pub mod command_config;
```

Comment:
command_config and helpers can be private

Reply:
I don't believe it